constants/errors.go

```diff
@@ -36,11 +36,16 @@ var (
 	ErrCloseClosedSession             = errors.New("close closed session")
 	ErrClosedGroup                    = errors.New("group closed")
 	ErrEtcdGrantLeaseTimeout          = errors.New("timed out waiting for etcd lease grant")
+	ErrEtcdLeaseNotFound              = errors.New("etcd lease not found in group")
```

Comment:
Try to leave the errors in alphabetical order, and take the opportunity to fix the comment at the top of the file